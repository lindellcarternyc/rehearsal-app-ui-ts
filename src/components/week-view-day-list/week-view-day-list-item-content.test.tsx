import * as React from 'react'
import * as renderer from 'react-test-renderer'

import WeekViewDayListItemContent, { WeekViewDayListItemContentProps } from './week-view-day-list-item-content'

describe('WeekViewDayListItemContent', () => {
  it('renders without rehearsals', () => {
    const props: WeekViewDayListItemContentProps = {
      rehearsals: []
    }
    const content = renderer.create(<WeekViewDayListItemContent {...props} />)
    expect(content).toMatchSnapshot()
  })

  it('renders with rehearsals', () => {
    const props: WeekViewDayListItemContentProps = {
      rehearsals: [
        {time: '19:00 - 21:00', material: ['Act I: Scene 1']}
      ]
    }
    const content = renderer.create(<WeekViewDayListItemContent {...props} />)
    expect(content).toMatchSnapshot()
  })
})