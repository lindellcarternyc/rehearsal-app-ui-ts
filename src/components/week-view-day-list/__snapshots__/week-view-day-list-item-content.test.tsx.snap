// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`WeekViewDayListItemContent renders with rehearsals 1`] = `
<div
  className="ui bulleted list"
  role="list"
>
  <div
    className="item"
    data-value={undefined}
    onClick={[Function]}
    role="listitem"
  >
    19:00 - 21:00
  </div>
</div>
`;

exports[`WeekViewDayListItemContent renders without rehearsals 1`] = `
<p
  style={
    Object {
      "paddingLeft": "1rem",
    }
  }
>
  No Rehearsals Today
</p>
`;
